### PR TITLE
Feat: Enhanced Tour

### PR DESCRIPTION
**Pull request by darth-coder00**
### Describe your changes :

I worked on the enhancement of Tour feature, with appropriate and verified messaged to guide through

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-01-04 at 7 38 14 PM
Screenshot 2022-01-04 at 7 38 29 PM
Screenshot 2022-01-04 at 7 43 41 PM
Screenshot 2022-01-04 at 7 44 22 PM
Screenshot 2022-01-04 at 7 44 44 PM
Screenshot 2022-01-04 at 7 44 53 PM
Screenshot 2022-01-04 at 7 45 01 PM
Screenshot 2022-01-04 at 7 45 13 PM
Screenshot 2022-01-04 at 7 45 20 PM
Screenshot 2022-01-04 at 7 45 32 PM
Screenshot 2022-01-04 at 7 45 39 PM
Screenshot 2022-01-04 at 7 45 47 PM
Screenshot 2022-01-04 at 7 45 54 PM
Screenshot 2022-01-04 at 7 46 01 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t @Sachin-chaurasiya

### CODE REVIEW

**Comment by mithmatt** [comment]
This is amazing...

File: openmetadata-ui/src/main/resources/ui/src/components/tour/Tour.tsx [hunk 1]
```diff
@@ -82,14 +134,26 @@ const getSteps = (value: string) => {
         AppState.activeTabforTourDatasetPage = 2;
       },
       actionType: 'click',
-      content: 'Click on the profiler tab.',
-      position: [75, 255],
+      content: () => (
+        <p>
+          Click on the <strong>&quot;profiler&quot;</strong> tab.
```
Comment (shahsank3t) [inline openmetadata-ui/src/main/resources/ui/src/components/tour/Tour.tsx:139]: `profiler` should be `Profiler`

**Review by shahsank3t — COMMENTED** [review]
Just one nit comment, except that, this looks amazing!

**Review by Sachin-chaurasiya — APPROVED** [review]
LGTM

**Pull request merged**